internal/compiler/builtins.slint

```diff
@@ -501,6 +501,7 @@ export component NativeTableHeaderSection {
 export component NativeComboBox {
     in-out property <string> current_value;
     in property <bool> enabled: true;
+    in property <bool> has-focus;
```

Comment:
Apparently it needs to be `out`  (as the test says)
But then you might not be able to set it from the ComboBox.
I need to think how best to solve this.